lib/models/device.js

```diff
@@ -779,6 +779,51 @@ const getDeviceModel = function (deps, opts) {
 					return ips.filter((ip) => ip !== vpn_address);
 				}),
 
+		/**
+		 * @summary Get the MAC addresses of a device
+		 * @name getMACAddresses
+		 * @public
+		 * @function
+		 * @memberof balena.models.device
+		 *
+		 * @param {String|Number} uuidOrId - device uuid (string) or id (number)
+		 * @fulfil {String[]} - mac addresses
+		 * @returns {Promise}
+		 *
+		 * @example
+		 * balena.models.device.getMACAddresses('7cf02a6').then(function(macAddresses) {
+		 * 	macAddresses.forEach(function(mac) {
+		 * 		console.log(mac);
+		 * 	});
+		 * });
+		 *
+		 * @example
+		 * balena.models.device.getMACAddresses(123).then(function(macAddresses) {
+		 * 	macAddresses.forEach(function(mac) {
+		 * 		console.log(mac);
+		 * 	});
+		 * });
+		 *
+		 * @example
+		 * balena.models.device.getMACAddresses('7cf02a6', function(error, macAddresses) {
+		 * 	if (error) throw error;
+		 *
+		 * 	macAddresses.forEach(function(mac) {
+		 * 		console.log(mac);
+		 * 	});
+		 * });
+		 */
+		getMACAddresses: (uuidOrId) =>
+			exports
+				.get(uuidOrId, { $select: ['mac_address'] })
+				.then(({ mac_address, ip_address, vpn_address }) => {
```

Comment:
`ip_address` and `vpn_address` is not used right?

Reply:
Indeed @iamsolankiamit .
I was pretty sure I deleted those :thinking: